README.md

```diff
@@ -38,3 +47,22 @@ Handwrite can generate custom font of your handwriting based on a writing sample
 `python setup.py test`
 
 You are ready to go!
+
+# Usage
+
+## Creating your Handwritten Sample 
+
+1. Take a printout of the [sample form](https://github.com/cod-ed/handwrite/raw/main/handwrite_sample.pdf).
+2. Fill the form using the image below as a reference.
+3. Scan the filled form using a scanner, or Adobe Scan in your phone.
+4. Save the `.jpg` image in your system.
+
+Your form should look like this:
+![Filled form](https://raw.githubusercontent.com/cod-ed/assets/handwrite/handwrite_filled_form.jpg)
```

Comment:
Can generate a font with this image and add a ss or gif after the Creating your font section? Will look nice.